golem_messages/message/concents.py

```diff
@@ -147,6 +147,76 @@ class FileInfo(datastructures.FrozenDict):
         }
 
 
+class ForceGetTaskResult(base.Message):
+    TYPE = CONCENT_MSG_BASE + 6
+
+    __slots__ = [
+        'report_computed_task',
+        'force_report_computed_task',
+    ] + base.Message.__slots__
+
+    def deserialize_slot(self, key, value):
+        value = super().deserialize_slot(key, value)
+        value = tasks.deserialize_report_computed_task(key, value)
+        value = deserialize_force_report_computed_task(key, value)
+        return value
+
+
+class ForceGetTaskResultAck(base.Message):
+    TYPE = CONCENT_MSG_BASE + 7
```

Comment:
@rwrzesien ah, and also, please update these `TYPE`s so that they don't collide with the other batch of newly-added messages that I have merged today ... 

(yeah, I know it's pain - I guess we could re-think the way these are assigned)